src/commands/Utility/languagestats.ts

```diff
@@ -28,7 +28,7 @@ const command: Command = {
 			discordLocale = transformDiscordLocale(interaction.locale)
 
 		let rawLang = interaction.options.getString("language", false)?.toLowerCase()
-		if ([!"en", "empty"].includes(authorDb.lang!) && discordLocale !== "en") rawLang ??= authorDb.lang ?? discordLocale
+		if (!["en", "empty"].includes(authorDb.lang ?? discordLocale)) rawLang ??= authorDb.lang ?? discordLocale
```

Comment:
Second part was intentional
```suggestion
		if (!["en", "empty"].includes(authorDb.lang) && discordLocale !== "en") rawLang ??= authorDb.lang ?? discordLocale
```

Reply:
Actually nvm